Resources/doc/cookbook/recipe_persisting_filters.rst

```diff
@@ -0,0 +1,138 @@
+
+You can implement your own filter persister by creating a new class that implements the
+``Sonata\AdminBundle\Filter\Persister\FilterPersisterInterface`` interface and registering it as a service.
+Then the only thing to do is to tell SonataAdmin to use this service as filter persister.
+
+
+Globally :
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
+        sonata_admin:
+            persist_filters: true
+            filter_persister: filter_persister_service_id
+
+
+Per Admin (using XML) :
+
+.. configuration-block::
+
+    .. code-block:: xml
+
+        <!-- src/AppBundle/Resources/config/admin.xml -->
+
+        <service id="app.admin.user" class="AppBundle\Admin\UserAdmin">
+            <tag name="sonata.admin" manager_type="orm" filters_persister="filter_persister_service_id" />
+            <argument />
+            <argument>AppBundle\Entity\User</argument>
+            <argument />
+        </service>
+
+
+Per Admin (using YAML) :
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # src/AppBundle/Resources/config/admin.yml
+
+        services:
+            app.admin.user:
+                class: AppBundle\Admin\UserAdmin
+                tags:
+                    -
+                        name: sonata.admin
+                        manager_type: orm
+                        filters_persister: filter_persister_service_id
+                arguments:
+                    - null
+                    - AppBundle\Entity\User
+                    - null
+
+
+Disable filters persistence for some Admin
+------------------------------------------
+
+When you enable the filters persistence by setting ``persist_filters`` to ``true``.
+All registered Admins will have the feature enabled.
+
+You can disable it per Admin if you want.
+
+
+Using XML :
+
+.. configuration-block::
+
+    .. code-block:: xml
+
+        <!-- src/AppBundle/Resources/config/admin.xml -->
+
+        <service id="app.admin.user" class="AppBundle\Admin\UserAdmin">
+            <tag name="sonata.admin" manager_type="orm" persist_filters="false" />
+            <argument />
+            <argument>AppBundle\Entity\User</argument>
+            <argument />
+        </service>
+
+
+Using YAML :
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # src/AppBundle/Resources/config/admin.yml
+
+        services:
+            app.admin.user:
+                class: AppBundle\Admin\UserAdmin
+                tags:
+                    -
+                        name: sonata.admin
+                        manager_type: orm
+                        persist_filters: false
+                arguments:
+                    - null
+                    - AppBundle\Entity\User
+                    - null
+
+
+.. note::
+
+    Both ``persist_filters`` and ``filter_persister`` can be used globally and per-admin.
+    Which provide you the most flexible way to configure this feature.
```

Comment:
This last sentence looks a bit weird, because it's a separate sentence and it's unclear what you refer to.